css/main.css

```diff
@@ -97,12 +97,39 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+#meetups-container ul {
+  list-style: none;
+  overflow: hidden;
+  margin-left: none;
+  padding-left: none;
+}
 
+#meetups-container ul li {
+  width: 230px;
+  padding: 15px;
+  margin-left: 25px;
+  height: 265px;
+  border-radius: 4px;
+  background-color: #efefef;
+  text-align: center;
+  overflow: hidden;
+  float: left;
+}
 
+#meetups-container ul li div.event-name{
```

Comment:
Wow that is a specific css selector. `#meetups-container .event-name` not good enough?